packages/vscode-extension/src/treeview/deploymentView.tsx

```diff
@@ -80,6 +80,14 @@ export class DeploymentView extends React.Component<any, any> {
           disable={this.state.locked}
           command="fx-extension.publish"
         ></TreeItem>
+        <TreeItem
+          label="Add CI/CD Workflows"
```

Comment:
why not reference the "addCICDWorkflowsTitle" in string.json as well?

Reply:
Good point, but I would not do this right now since that would be inconsistent if I reference string.json here but other items in the same level are not. Better to change them all in a batch, and I would like to leave this to later contributors to save efforts for this PR.

Reply:
Actually the deploymentView here is not used in production code, so it's OK to keep it.